portal/translations/es_ES/LC_MESSAGES/messages.po

```diff
@@ -2276,186 +2294,190 @@ msgid "America/New York"
 msgstr "Estados Unidos/Nueva York"
 
 #: https://github.com/uwcirg/truenth-portal/tree/develop/portal/templates/profile/profile_macros.html#L932
+msgid "America/Sao Paulo"
+msgstr ""
```

Comment:
@ivan-c  just noting this new msgid and it's empty msgstr

Reply:
Probably not translated (authorized) yet; I'll make sure to add it when I go through the strings needing authorization